yarn.lock

```diff
@@ -24016,15 +24016,15 @@ react-sizeme@^2.5.2, react-sizeme@^2.6.7:
     shallowequal "^1.1.0"
     throttle-debounce "^2.1.0"
 
-react-sizeme@^2.6.3:
-  version "2.6.3"
-  resolved "https://registry.yarnpkg.com/react-sizeme/-/react-sizeme-2.6.3.tgz#82659d64249c0bf8fc1ddc60b4965d3fb58a5680"
+react-sizeme@^3.0.1:
```

Comment:
Why did react-sizeme get bumped here?

Reply:
after checking the various card/dashboard editor/hotspot modal stories, it doesn't seem like it breaks anything, but a second set of eyes would be nice.

Reply:
I don't know what caused this change, I only ran `yarn install` and `yarn test -u` before committing the code.

Reply:
interesting. this might be a hold-over from the recent carbon upgrade. @jkeen, did you run into any issues with react-sizeme being bumped?

Reply:
please let me know if you want me to revert the change in `yarn.lock`.